projects/eigen/solver_fuzzer.cc

```diff
@@ -0,0 +1,57 @@
+#include <cstddef>
```

Comment:
I think this file needs a license/copyright header.

Reply:
All code (ie: build.sh, c++, and Dockerfile) checked into oss-fuzz needs one of these.